src/Simulation/Simulators.Tests/Circuits/ExpTest.qs

```diff
@@ -27,6 +28,28 @@ namespace Microsoft.Quantum.Simulation.Simulators.Tests.Circuits {
         }
     }
     
+    operation VerifyExpUsingDecompositions() : Unit {
```

Comment:
This makes perfect sense to add, fully agreed. For the benefit of future maintainers, would it help to add a comment that this is mainly useful for checking the magnitude of the angle and not its sign?

Reply:
Good call, adding comment.